### PR TITLE
Loader issue fix and added webview lifecycle

### PR DESCRIPTION
**Pull request by tarun-gujjula**

### CODE REVIEW

File: VerloopSDK/VLViewController.swift [hunk 1]
```diff
@@ -75,8 +84,32 @@ class VLViewController: UIViewController, WKUIDelegate {
...
         }
-        self.updateWebViewConstraints()
+        loader = UIActivityIndicatorView(frame: CGRect(x: view.center.x, y: view.center.y, width: 30, height: 30))
+        loader?.tintColor = .black
+        if #available(iOS 13.0, *) {
+            loader?.style = .large
+        }
+        loader?.center = self.view.center
+        loader?.startAnimating()
+        view.addSubview(loader!)
```
Comment (tarun-gujjula) [inline VerloopSDK/VLViewController.swift:96]: this will crash if loader is nil

File: VerloopSDK/VerloopSDK.swift [hunk 2]
```diff
@@ -145,7 +146,7 @@ import Foundation
             verloopNavigationController?.view.backgroundColor = .white
 
             verloopNavigationController?.navigationBar.barTintColor = UIColor.white
-
+            verloopController?.setViewControllerLifeCycleDelegate(manager)
```
Comment (tarun-gujjula) [inline VerloopSDK/VerloopSDK.swift:149]: You could've directly set the manager as delegate in the VLViewController initialisation, and not as a seperate method.

File: VerloopSDK/VLViewController.swift (hunk 1 above)
Comment (sreedeep-ios) [reply VerloopSDK/VLViewController.swift:96]: when you are creating references that can be optional but when we are adding a view as subview that cannot be optional and we have added this instance just above so there is no possibility of this being nil.

File: VerloopSDK/VerloopSDK.swift (hunk 2 above)
Comment (sreedeep-ios) [reply VerloopSDK/VerloopSDK.swift:149]: done

**Pull request merged**